rcon/steam_utils.py

```diff
@@ -15,6 +17,39 @@
 logger = logging.getLogger(__name__)
 
 last_steam_api_key_warning = datetime.datetime.now()
+steam_id_64_pattern = re.compile(r"\d{17}")
+
+
+def is_steam_id_64(steam_id_64: str) -> bool:
+    if re.match(steam_id_64_pattern, steam_id_64):
+        return True
+    return False
+
+
+def filter_steam_ids():
```

Comment:
I think for the time being, this is good enough for now. It abstracts away the explicit check if a player ID is a steam ID, however, for now that's fine.

I would wish that there will be _any_ way to get similar information about a player profile for a Windows Store version of HLL, but I doubt that we will get this in the near future or at all.

Reply:
I've inquired but I doubt that we're ever going to have an API we can inquire against. It's not clear to me, but I believe these IDs are being generated by Team17 from some sort of windows store identifier :<

Reply:
> I believe these IDs are being generated by Team17 from some sort of windows store identifier

That's exactly what I got from the document shared by T17 as well 🙈